### DISCUSSION TITLE
Targeting Type in Ads API

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/ads-advanced-tools-docs#271] [Q&A]
Hi

Could you please tell me which API provides information on Targeting Types( Manual targeting and Automatic targeting) types for a campaign? I couldn't find it in the documentation.

**Comment by elizafarley** [comment]
Hi @Nilanth, the auto vs. manual targeting distinction is only relevant for Sponsored Products campaigns. You can use [POST /campaigns/export](https://advertising.amazon.com/API/docs/en-us/exports#tag/Exports/operation/CampaignExport) and check the `targetingSettings` field to see the value associated to the campaign.

**Reply by Nilanth** [reply]
Hi @elizafarley , I checked the API you have mentioned. But there is not key named like  **targetingSettings**.

**Reply by Sourabh-jana** [reply]
Hey @Nilanth you will find this in here: [https://advertising.amazon.com/API/docs/en-us/reference/common-models/campaigns#json-examples](url)

copy and paste this url and check the fields coming.
